src/main/java/org/broadinstitute/hellbender/Main.java

```diff
@@ -58,6 +60,19 @@
     private static final String STACK_TRACE_ON_USER_EXCEPTION_PROPERTY = "GATK_STACKTRACE_ON_USER_EXCEPTION";
 
     /**
+     * Prints the given message (may be null) to the provided stream, adding adornments and formatting.
+     */
+    public static void printDecoratedUserExceptionMessage(final PrintStream ps, final Exception e){
+        Utils.nonNull(ps, "stream");
+        Utils.nonNull(e, "exception");
+        ps.println("***********************************************************************");
+        ps.println();
+        ps.println("A USER ERROR has occurred: " + e.getMessage());
```

Comment:
If someone extends main and want to handle some exceptions in a different way, decorating the exception may be useful as public or protected. And if the preamble is configurable, it may help to handle other exceptions differently for other toolkit.